src/default-value.coffee

```diff
@@ -0,0 +1,56 @@
+# Generate default values based on an element type and its path in the
+# element tree. This method tries to be smart about generating a sample data
+# structure that makes sense when no samples or defaults were given.
+faker = require 'faker'
+
+endsWith = (input, search) ->
+  if not input then return false
+  position = input.length - search.length
+  lastIndex = input.indexOf search, position
+  lastIndex isnt -1 and lastIndex is position
+
+module.exports = (item, path) ->
```

Comment:
_**Nit:**_ If I'm not mistaken `item` here is a Refract element? Perhaps `refractElement` would be more clear?